docs/README.md

```diff
@@ -2,95 +2,103 @@
-Crossplane extends a Kubernetes cluster to support orchestrating any infrastructure or managed service. Providers extend
-Crossplane to enable infrastructure resource provisioning of specific API.
+Crossplane Provider for IONOS Cloud gives the ability to manage IONOS Cloud infrastructure directly from Kubernetes. Crossplane extends a Kubernetes cluster to support orchestrating any infrastructure or managed service. Providers can use Crossplane to enable infrastructure resource provisioning of a specific API.
 
-The [crossplane-provider-ionoscloud](https://github.com/ionos-cloud/crossplane-provider-ionoscloud) repository is the
-Crossplane infrastructure provider for IONOS Cloud. The provider that is built from the source code from the repository
-can be installed into a Crossplane control plane and adds the following new functionality:
+{% hint style="info" %}
+**Note:** You can access and contribute to the [<mark style="color:blue;">crossplane-provider-ionoscloud</mark>](https://github.com/ionos-cloud/crossplane-provider-ionoscloud) repository for IONOS Cloud.
+{% endhint %}
 
-* Custom Resource Definitions (CRDs) that model IONOS Cloud infrastructure and services (e.g. Compute Engine,
-  Kubernetes, Database As a Service Postgres, etc.)
-* Controllers to provision these resources in IONOS Cloud based on the users desired state captured in CRDs they create
-* Implementations of Crossplane's portable resource abstractions, enabling IONOS Cloud resources to fulfill a user's
-  general need for cloud services
+ The provider that is built from the source code from the repository can be installed into a Crossplane control plane and adds the following new functionality:
 
-## Getting Started and Documentation
+  * Custom Resource Definitions (CRDs) model IONOS Cloud infrastructure and services. For example, Compute Engine, Kubernetes, Database As a Service Postgres, etc.
+  * Controllers provision these resources in IONOS Cloud based on the user's desired state captured in CRDs when created.
+  * Implementations of Crossplane's portable resource abstractions that enable IONOS Cloud resources to fulfill a user's general need for cloud services.
 
-For getting started with Crossplane usage and concepts, see the
-official [Documentation](https://crossplane.io/docs/v1.7/).
+## Getting started 
 
-For getting started with Crossplane Provider for IONOS Cloud, check out this
-step-by-step [guide](../examples/example.md) which provides details about the provisioning of a Postgres Cluster in
-IONOS Cloud.
+To start with Crossplane usage and concepts, see the official [<mark style="color:blue;">Crossplane Documentation</mark>](https://docs.crossplane.io/v1.15/getting-started/).
+
+To get started with Crossplane Provider for IONOS Cloud, see [<mark style="color:blue;">Crossplane Provider IONOS Cloud POC</mark>](../examples/example.md), which provides details about the provisioning of a **Postgres cluster** in IONOS Cloud.
```

Comment:
I added another comment to explain this change

```suggestion
To get started with Crossplane Provider for IONOS Cloud, see [<mark style="color:blue;">Crossplane Provider IONOS Cloud Usage Example</mark>](../examples/example.md), which provides details about the provisioning of a **Postgres cluster** in IONOS Cloud.
```

Reply:
Done, thanks a lot.